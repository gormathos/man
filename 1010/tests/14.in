sol_id;user_id;task_id;lang_id;score
139745;1018;2003;1;0.00
139746;1096;2005;2;16.00
139748;1085;2005;4;80.00
139751;1093;2005;3;8.00
139755;1077;2005;1;100.00
139760;1049;2005;3;100.00
139766;1061;2002;2;0.00
139773;1071;2001;5;0.00
139781;1061;2002;4;0.00
139790;1011;2004;3;54.55
139800;1019;2002;3;0.00
139811;1048;2001;2;57.14
139823;1012;2005;4;92.00
139836;1041;2002;3;51.35
139850;1062;2003;4;0.00
139865;1054;2004;3;0.00
139881;1053;2003;4;69.77
139898;1082;2003;1;72.09
139916;1001;2002;1;100.00
139935;1057;2004;4;0.00
139955;1092;2005;1;0.00
139976;1061;2005;4;0.00
139998;1082;2005;2;0.00
140021;1067;2004;3;72.73
140045;1071;2005;1;100.00
140070;1021;2004;2;0.00
140096;1081;2002;2;100.00
140123;1092;2004;2;100.00
140151;1053;2004;4;27.27
140180;1077;2005;2;32.00
140210;1045;2005;3;48.00
140241;1010;2004;1;90.91
140273;1037;2003;4;11.63
140306;1048;2005;1;100.00
140340;1008;2001;4;100.00
140375;1009;2001;1;0.00
140411;1095;2005;4;84.00
140448;1093;2001;1;0.00
140486;1071;2004;3;13.64
140525;1091;2002;3;51.35
140565;1096;2001;1;100.00
140606;1079;2002;3;51.35
140648;1015;2004;1;100.00
140691;1100;2004;5;54.55
140735;1050;2004;3;100.00
140780;1090;2002;5;0.00
140826;1008;2003;2;100.00
140873;1034;2004;2;100.00
140921;1017;2003;3;100.00
140970;1039;2002;5;100.00
141020;1072;2001;2;0.00
141071;1029;2003;3;62.79
141123;1014;2001;5;100.00
141176;1094;2002;3;8.11
141230;1032;2003;1;4.65
141285;1059;2001;3;100.00
141341;1006;2005;1;60.00
141398;1038;2005;3;0.00
141456;1097;2004;1;50.00
141515;1004;2001;3;100.00
141575;1059;2002;1;24.32
141636;1038;2004;3;0.00
141698;1063;2002;5;0.00
141761;1026;2002;1;100.00
141825;1089;2004;2;27.27
141890;1026;2001;4;0.00
141956;1089;2002;3;100.00
142023;1041;2005;5;16.00
142091;1069;2005;4;72.00
142160;1019;2005;2;100.00
142230;1005;2004;4;100.00
142301;1076;2001;2;67.86
142373;1073;2005;4;12.00
142446;1025;2002;4;0.00
142520;1087;2002;2;10.81
142595;1065;2004;2;100.00
142671;1024;2001;5;14.29
142748;1079;2003;2;20.93
142826;1078;2002;5;51.35
142905;1059;2004;2;63.64
142985;1094;2001;1;100.00
143066;1037;2001;1;100.00
143148;1017;2002;5;100.00
143231;1030;2004;5;100.00
143315;1020;2005;2;0.00
143400;1080;2005;3;0.00
143486;1007;2002;1;100.00
143573;1096;2004;5;72.73
143661;1063;2003;2;100.00
143750;1036;2002;4;18.92
143840;1050;2001;4;25.00
143931;1033;2004;5;31.82
144023;1002;2002;5;0.00
144116;1081;2002;2;13.51
144210;1090;2003;3;13.95
144305;1080;2005;5;12.00
144401;1026;2004;5;81.82
144498;1032;2004;1;0.00
144596;1030;2002;2;8.11
144695;1075;2003;1;67.44
144795;1100;2004;4;100.00
144896;1001;2003;3;95.35
144998;1028;2001;2;100.00
145101;1077;2001;2;14.29
145205;1018;2005;1;100.00
145310;1056;2004;5;50.00
145416;1081;2004;3;4.55
145523;1062;2004;4;0.00
145631;1028;2005;4;0.00
145740;1054;2004;3;100.00
145850;1026;2004;2;0.00
145961;1021;2002;2;70.27
146073;1051;2003;1;44.19
146186;1056;2004;5;100.00
146300;1079;2005;3;12.00
146415;1008;2002;2;100.00
146531;1060;2004;1;100.00
146648;1087;2001;1;35.71
146766;1027;2004;4;0.00
146885;1007;2005;4;24.00
147005;1061;2005;1;0.00
147126;1064;2004;5;100.00
147248;1012;2004;3;50.00
147371;1028;2005;4;100.00
147495;1006;2001;5;46.43
147620;1092;2004;4;68.18
147746;1026;2002;4;100.00
147873;1070;2003;2;86.05
148001;1049;2003;2;0.00
148130;1068;2002;2;100.00
148260;1028;2003;3;0.00
148391;1009;2005;5;36.00
148523;1060;2001;2;71.43
148656;1021;2005;3;0.00
148790;1057;2001;5;60.71
148925;1040;2004;4;100.00
149061;1073;2002;2;100.00
149198;1028;2005;1;60.00
149336;1031;2005;2;88.00
149475;1049;2001;4;96.43
149615;1076;2002;5;0.00
149756;1066;2004;4;50.00
149898;1023;2002;4;0.00
150041;1100;2004;1;31.82
150185;1050;2002;3;0.00
150330;1096;2003;1;100.00
150476;1054;2004;5;100.00
150623;1047;2003;1;9.30
150771;1061;2002;4;37.84
150920;1039;2003;1;0.00
151070;1043;2001;4;100.00
151221;1050;2005;2;100.00
151373;1032;2005;2;0.00
151526;1084;2003;5;100.00
151680;1075;2001;5;85.71
151835;1055;2004;1;0.00
