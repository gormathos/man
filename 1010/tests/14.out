Pascal: 30 33 13 8
Python: 31 35 12 8
C++: 26 30 7 8
Java: 26 32 7 10
C#: 25 26 8 5
